wd_helper.au3

```diff
@@ -777,7 +747,7 @@ Func _WD_LoadWait($sSession, $iDelay = Default, $iTimeout = Default, $sElement =
 	WEnd
 
 	If $iErr Then
-		Return SetError(__WD_Error($sFuncName, $iErr, ""), 0, 0)
+		Return SetError(__WD_Error($sFuncName, $iErr), $_WD_HTTPRESULT, 0)
```

Comment:
`@extended` should or should not be set to `$_WD_HTTPRESULT`?

Reply:
I thought this was already decided.

> We probably should eliminate setting @Extended = $_WD_HTTPRESULT and have them call _WD_LastHTTPResult instead when needed.

Reply:
I know this ones I must to revert back.

Reply:
> I know this ones I must to revert back/clean up.

Already done